libraries/src/Updater/Update.php

```diff
@@ -690,4 +700,33 @@ protected function stabilityTagToInteger($tag)
 
         return Updater::STABILITY_STABLE;
     }
+
+    /**
+     * Set extension's Joomla! target version
+     *
+     * @param   string  $version  The target version
+     *
+     * @return  void
+     *
+     * @since   __DEPLOY_VERSION__
+     */
+    public function setTargetVersion($version)
+    {
+        $this->targetVersion = $version;
+    }
+
+    /**
+     * Get extension's Joomla! target version
+     *
+     * @return  string
+     *
+     * @since   __DEPLOY_VERSION__
+     */
+    public function getTargetVersion()
+    {
+        if (!$this->targetVersion) {
+            return JVERSION;
+        }
+        return $this->targetVersion;
```

Comment:
```suggestion

        return $this->targetVersion;
```
One last code style thing.